Automattic/wp-calypso
# Paid Stats: Gate StatsModuleSummaryLinks for desktop

Fixes https://github.com/Automattic/dotcom-forge/issues/4986

## Proposed Changes

![Gated](https://github.com/Automattic/wp-calypso/assets/6586048/ae5b01b9-a412-4bff-8649-45b70a422bcc)

https://github.com/Automattic/wp-calypso/assets/6586048/f4620c11-8f27-4455-a27c-5d6eac48fac0



## Testing Instructions



* Go to /stats with `?flags=stats/paid-wpcom-v2` attached to the end of the url
* Click on "View Details" in `Posts & pages` or `Countries`
* Click on the Gated stats (should be gated for WPCom sites that hasn't purchased PWYW and not Jetpack sites)

## Pre-merge Checklist
